App crashes in AuthenticationActivity on pre-lollipop devices

```
java.lang.RuntimeException: Unable to start activity
ComponentInfo{org.amahi.anywhere/org.amahi.anywhere.activity.AuthenticationActivity}:
android.view.InflateException: Binary XML file line #52: 
Error inflating class android.support.design.widget.TextInputLayout

Caused by: java.lang.IllegalArgumentException: 
You need to use a Theme.AppCompat theme (or descendant) with the design library.
```

This can be solved by using AppCompatActivity #96

--- a/src/main/res/values/themes.xml
+++ b/src/main/res/values/themes.xml
@@ -19,7 +19,10 @@
   -->
 
 <resources>
-	<style name="Theme.Amahi.Authentication" parent="@style/Theme.Amahi">
-		<item name="android:windowNoTitle">true</item>
-	</style>
+    <style name="Theme.Amahi.Authentication" parent="Theme.AppCompat">
+        <item name="android:windowNoTitle">true</item>
+        <item name="android:actionBarStyle">@style/Theme.Amahi.ActionBar</item>
+        <item name="android:windowBackground">@color/background_primary</item>
+        <item name="android:textColorPrimaryInverse">@color/primary_text_material_light</item>
+    </style>
 </resources>

